functions/index.js

```diff
@@ -0,0 +1,9 @@
+// const functions = require("firebase-functions");
```

Comment:
Given that we are not going to be using Firebase Functions as part of this project, please don't check in a `functions/` directory.

Reply:
Done!

Reply:
The folder is still here in the PR